Bottom: c80b4be4ab92d132c1c4c9fd54bf99a29979d111
Top:    b9821977dcb40485583408d46e1ac8a95854c311
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-24 16:30:47 -0500

Remove libicu52 from Dockerfile

Newest upstream versions of php:5.6-apache are based on debian stretch and 
use libicu57, which is already installed. Besides this, libicu-dev would 
already install libicuXX.


---
diff --git a/docker/docker-mediawiki/Dockerfile b/docker/docker-mediawiki/Dockerfile
index 6aaefd61..54421165 100644
--- a/docker/docker-mediawiki/Dockerfile
+++ b/docker/docker-mediawiki/Dockerfile
@@ -13,7 +13,6 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         git \
         imagemagick \
-        libicu52 \
         libicu-dev \
         libldap2-dev \
     && \
